.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,25 @@
 <!--
-File an issue if you think you've found a bug. Please describe
 
-1. How can it be reproduced
-2. What was expected
-3. What actually occurred
-4. What version of the involved component
--->
\ No newline at end of file
+Template: BUG REPORT
+
+This is an issue for a bug report.
+You should label `bug-report`.
+
+
+If your purpose is different, please visit following links.
+
+FEATURE REQUEST:
+https://github.com/naokazuterada/MarkdownTOC/issues/new?template=feature.md
+
+QUESTION:
+https://github.com/naokazuterada/MarkdownTOC/issues/new?template=question.md
+
+-->
+
+# How can it be reproduced
+
+# What was expected
+
+# What actually occurred
+
+# What version of the involved component
```

Comment:
Suggestion: "What was the version of the involved component"

Reply:
👍